core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.controller.spec.ts

```diff
@@ -121,4 +121,22 @@ describe('Add Misconception Modal Controller', function() {
     $scope.saveMisconception();
     expect($uibModalInstance.close).not.toHaveBeenCalled();
   });
+
+  it('should check content length in misconception notes within 1000 chars',
```

Comment:
'should check if the content length of misconception notes is within 1000 chars'. Similarly, you may write for the next test.

Reply:
I intentionally shortened that sentence since it was going over the line word limit ( 80 words ) for eslint . Adding another word will make us shift the last word in next line . So just for confirmation , asking if you really want these changes or let it be as it is ?